exporters/css/src/files/style-file.ts

```diff
@@ -1,32 +1,55 @@
-import { FileHelper } from "@supernovaio/export-helpers"
-import { OutputTextFile, Token, TokenGroup, TokenType } from "@supernovaio/sdk-exporters"
-import { exportConfiguration } from ".."
-import { convertedToken } from "../content/token"
+import { FileHelper } from '@supernovaio/export-helpers';
+import {
+  OutputTextFile,
+  Token,
+  TokenGroup,
+  TokenType,
+} from '@supernovaio/sdk-exporters';
+import { exportConfiguration } from '..';
+import { convertedToken } from '../content/token';
 
-export function styleOutputFile(type: TokenType, tokens: Array<Token>, tokenGroups: Array<TokenGroup>): OutputTextFile | null {
+export function styleOutputFile(
+  type: TokenType,
+  tokens: Array<Token>,
+  tokenGroups: Array<TokenGroup>
+): OutputTextFile | null {
   // Filter tokens by top level type
-  const tokensOfType = tokens.filter((token) => token.tokenType === type)
+  const tokensOfType = tokens.filter((token) => token.tokenType === type);
+
+  // Only export supported token types
+  // TODO: Add all token types
+  const supportedTypes: Array<TokenType> = [TokenType.color];
```

Comment:
Begrenset slik at det nå kun genereres fil for color tokens, ikke typography, border++ som også er mulig å legge inn i Supernova. Tenker man kan legge inn typene litt etterhvert når man bruker de i supernova så man får sjekket at formatet som da genereres blir ønskelig